app.py

```diff
@@ -1,66 +1,264 @@
 import os
 from chalice import Chalice
-from chalicelib.lib import dynamo_lock, dynamo_event
-from chalicelib.model import Dynamo
+from chalicelib.lib import db_v1, db_v2
+from chalicelib.model.models import EventTable, LockTable
 import logging
 
 
 app = Chalice(app_name='timereport_backend')
 app.debug = os.getenv('BACKEND_DEBUG', False)
 log = logging.getLogger(__name__)
 
-db_event = Dynamo.EventModel
-db_lock = Dynamo.LockModel
 
-# create tables
-for db in [db_event, db_lock]:
-    if not db.exists():
-        db.create_table(read_capacity_units=1, write_capacity_units=1, wait=True)
+for db_instance in [EventTable, LockTable]:
+    if not db_instance.exists():
+        db_instance.create_table(read_capacity_units=1, write_capacity_units=1, wait=True)
```

Comment:
This capacity stuff might benefit from being possible to change (e.g load values from env variables if defined)

Reply:
Good idea, should create a separate ticket for it